scripts/anchor-links.js

```diff
@@ -0,0 +1,42 @@
+// @ts-check
+
+module.exports = function setupAnchorLinks () {
+  /**
+   * @param {string} id
+   */
+  function anchorForId (id) {
+    const anchor = document.createElement('a')
+    anchor.className = 'header-link'
+    anchor.href = '#' + id
+    anchor.innerHTML = "<span class='octicon octicon-link'></span>"
+    anchor.title = 'Permalink'
+    return anchor
+  };
+
+  /**
+   * @param {string | number} level
+   * @param {Element} containingElement
+   */
+  function linkifyAnchors (level, containingElement) {
+    const headers = containingElement.getElementsByTagName('h' + level)
+    for (let h = 0; h < headers.length; h++) {
+      const header = headers[h]
+
+      if (typeof header.id !== 'undefined' && header.id !== '') {
+        header.appendChild(anchorForId(header.id))
+      }
+    }
+  };
+
+  document.onreadystatechange = function () {
+    if (this.readyState === 'complete') {
```

Comment:
I know this was just moved from the partial, but since we're here, can we change this to use `addEventListener`?

```javascript
  document.addEventListener('readystatechange', () => {
    if (document.readyState === 'complete') {
```

Reply:
Done in bf85e1527